examples/02_07_specific_words_with_tfidf.ipynb

```diff
@@ -0,0 +1,198 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "WGwuELZZUfED"
+   },
+   "source": [
+    "# Iskanje besed, specifičnih za dokumente z uporabo transformacije TF-IDF"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "pmEW2fr2Up_2"
+   },
+   "source": [
+    "Tokrat bomo poskusili specifične besede v dokumentu določiti z uporabo transformacije TF-IDF, ki uteži besede glede na njihovo pogostost v besedilu. Besede, ki močno zaznamujejo manjšo množico dokumentov, bodo tako imele večjo težo kot take, ki so vseprisotne v korpusu."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "FeLVMjMLWVhR"
+   },
+   "source": [
+    "Iz seznama predlogov vladi izberemo zadnjih 100 dokumentov, jih predobledamo na enak način kot v prejšnjih primerih in za vsak dokument izračunamo najbolj specifične besede. Uporabili bomo utežitev TF-IDF in poiskali nekaj najvišje uteženih besed v množici."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {
+    "id": "E7oBUANqIGzm"
+   },
+   "outputs": [],
+   "source": [
+    "from textsemantics.server_api import ServerAPI\n",
+    "import string\n",
+    "import nltk\n",
+    "from nltk.tokenize import RegexpTokenizer\n",
+    "from nltk.corpus import stopwords\n",
+    "from lemmagen.lemmatizer import Lemmatizer\n",
+    "from lemmagen import DICTIONARY_SLOVENE\n",
+    "from flair.data import Sentence\n",
+    "from flair.embeddings import WordEmbeddings, DocumentPoolEmbeddings\n",
+    "from sklearn.feature_extraction.text import TfidfVectorizer\n",
+    "import numpy as np\n",
```

Comment:
flair imports and NumPy import are not required

Reply:
thanks, apparently a remnant of the copy-paste 😊